conf/pf.conf.defaults

```
@@ -757,6 +757,31 @@ pfcmd_warning_color=yellow
 #
 # Color of the success text for pfcmd
 pfcmd_success_color=green
+#
+# advanced.use_omapi_to_lookup_mac
+#
+# Use omapi to query dhcpd for the mac address of an ip
+use_omapi_to_lookup_mac=disabled
```

In this case it will simply silently fail without any impact I think... no ?